test/colony-network-mining.js

```diff
@@ -314,9 +316,24 @@ contract("ColonyNetworkMining", accounts => {
       assert.isTrue(nSubmittedHashes.isZero());
     });
 
-    it("should not allow someone to withdraw their stake if they have submitted a hash this round", async () => {
-      await giveUserCLNYTokensAndStake(colonyNetwork, MAIN_ACCOUNT, DEFAULT_STAKE);
+    it("should not allow someone to submit a new reputation hash if they stake after the cycle beings", async () => {
+      await forwardTime(1);
```

Comment:
Why this?

Reply:
I added a comment, basically the conditional is `windowOpen >= stakeTimestamp` (done so the staking in migration 8 works), which made the tests fail randomly since sometimes the cycle was advanced in the same block as the staking occurred and so the timestamps were equal. This ensures that the staking happens strictly after the window opens.